cmake/test.cmake

```diff
@@ -5,9 +5,16 @@ if(MayaUsd_FOUND)
     # Get the current value of the environment variable
     set(CURRENT_MAYA_MODULE_PATH $ENV{MAYA_MODULE_PATH})
     # Append the new path to the current value
-    set(MAYA_MODULE_PATH "${CURRENT_MAYA_MODULE_PATH}:${MAYAUSD_MOD_PATH}")
+    if(IS_MACOSX OR IS_LINUX) 
+        #Linux and OSX
+        set(MAYA_MODULE_PATH "${CURRENT_MAYA_MODULE_PATH}:${MAYAUSD_MOD_PATH}")
+    else()
+        #Windows
+        set(MAYA_MODULE_PATH "${CURRENT_MAYA_MODULE_PATH};${MAYAUSD_MOD_PATH}")
```

Comment:
Sorry I should have remembered this earlier. The ; needs to be escaped otherwise cmake will interpret it to mean the string is a list. This is documented on line 78 in this file and used that way on line 343.